azuredeploy.json

```diff
@@ -187,6 +187,18 @@
                             "echo \"Completed apt-get update/dist-upgrade\""
                         ]
                     },
+                    {
+                        "type": "Shell",

```

Comment:
If I'm understanding the goal in here, I'm not so sure running scripts to set policies is a good recommendation at the organization level.

Should we add some comment from the narrative?

Reply:
However I'm not an expert on how to manage them, honestly. So this is just a shot in the dark...

Reply:
I would love we could set this like using AD or similar.

Reply:
Thanks for that.  Let me see if I understand your concern.

I'm hoping the fact that you should secure jump boxes (or any image) to the level necessary is already covered in the guidance on this page.  We have a section that addresses hardening, and that this image isn't hardened.  However, I don't quite follow your guidance recommendation around making sure that your initial image shouldn't be hardened based on every available option to you.  Waiting until a security agent makes changes on your machine AFTER you boot it would cause an additional lapse in coverage.  For example, doing it via cloud-init or group policies.  That said, those are both great options, and can be applied as such as well.

This image isn't implied to be connected to Azure AD nor is it implied that it will be booted within a cloud-init context.  Does that help at all?

Reply:
thanks for sharing this! It definitely helps me understand the end goal better and I do like it. 

You are right, this RI is solely about image building. I was just going beyond the boundaries for this by asking for an after boot Policy Management, which is what I was envisioning we should recommend instead.  

Perhaps this is something to mention as part of regulated (not actually connected to anything, but how it would be IRL).